ovs/extensions/hypervisor/apis/vmware/sdk.py

```diff
@@ -903,13 +903,88 @@ def get_nfs_datastore_object(self, ip, mountpoint, filename, host=None):
                     return vm, mapping[datastore.name][filename]
         raise RuntimeError('Could not locate given file on the given datastore')
 
+    @authenticated()
+    def create_disk(self, ip, mountpoint, disk_path, size, host=None, wait=True):
+        if host is None:
+            host = self._esxHost
```

Comment:
This if-statement is not required, you can just pass in host into ```self._validate_host```, even if it's None. It will be filled with ```self._esxHost``` then.